jupyter/run-jupyter.sh

```diff
@@ -16,6 +16,17 @@ ln -snf "${JUPYTER_CONFIG_DOCKER}" $HOME/.jupyter
 
 rm /app/.local/share/jupyter/runtime/jpserver-*-open.html 2>/dev/null
 
+: ${HOST:=localhost}
+SSLPFX="/app/work/docker_volume/${HOST}"
+if [[ (! -f ${SSLPFX}-key.pem) || (! -f ${SSLPFX}-cert.pem) ]]; then
+    openssl req -batch -x509 -newkey rsa:4096 -days 10000 \
+    -subj "/C=US/ST=California/O=Aryn.ai/CN=${HOST}" \
+    -extensions v3_req -addext "subjectAltName=DNS:${HOST}" \
+    -noenc -keyout "${SSLPFX}-key.pem" -out "${SSLPFX}-cert.pem" \
+    2> /dev/null
```

Comment:
We should check for an error/save the logs in the openssl call. The same || die construction from sycamore-opensearch.sh (and copy over die)